pyclam/chaoda.py

```diff
@@ -214,21 +188,21 @@ def random_walk(graph: Graph, normalize: str = 'gaussian') -> np.array:
 
     # normalize counts
     scores: Dict[int, int] = {point: -count for cluster, count in visit_counts.items() for point in cluster.argpoints}
-    return _normalize(np.array([scores[i] for i in range(graph.population)], dtype=int), mode=normalize)
+    return normalize(np.array([scores[i] for i in range(graph.population)], dtype=int), mode=normalization_mode)
 
 
-def stationary_probabilities(graph: Graph, normalize: str = 'gaussian') -> np.array:
+def stationary_probabilities(graph: Graph, normalization_mode: str = 'gaussian') -> np.array:
```

Comment:
I think we should use this: https://docs.python.org/3/library/typing.html#typing.Literal

Reply:
I think we should have a separate PR for using literals. I think we should treat this more like we treat Metric. some strings to represent build-in functionality, while also allowing a user-defined function. In either case, I think that's outside the scope of this PR